estuary/utils/recents.py

```diff
@@ -29,21 +28,17 @@ def get_recent_nodes():
         Advisory.__label__: 'update_date'
     }
 
-    query = 'CALL apoc.cypher.runMany(\''
+    final_result = {}
     for label, time_property in label_dict.items():
-        query += (
-            '\n    MATCH ({label}:{label}) RETURN {label} '
+        query = (
+            'MATCH ({label}:{label}) RETURN {label} '
```

Comment:
Can you call the variable name something generic? It's a bit confusing that `label` is used everywhere. We did that before because `runMany` needed distinct variables for each query, but we don't need to anymore. You could do something like `item` instead.

Reply:
Just to confirm, change all three to `item` ?